client/.env.development.local

```diff
@@ -1,3 +1,3 @@
-REACT_APP_API_ROOT=http://localhost:3001
+REACT_APP_API_ROOT=https://testnet-api.casperdash.io
```

Comment:
hi please revert it. Let's use `yarn dev-extension-testnet` if you want connect to testnet API.

Reply:
Updated